code/game/objects/structures/deployable_turret.dm

```diff
@@ -213,7 +213,7 @@
 
 /obj/machinery/deployable_turret/hmg
 	name = "heavy machine gun turret"
-	desc = "A heavy calibre machine gun commonly used by Nanotrasen forces, famed for it's ability to give people on the recieving end more holes than normal."
+	desc = "A heavy caliber machine gun commonly used by Nanotrasen forces, famed for it's ability to give people on the receiving end more holes than normal."
```

Comment:
'Calibre' is not an incorrect spelling. 'Caliber' is the American English spelling, while 'Calibre' is the British English spelling.

Reply:
![image](https://github.com/tgstation/tgstation/assets/69338705/48bb20b4-86a7-40fa-afb2-00ab5b867736)

![image](https://github.com/tgstation/tgstation/assets/69338705/582e083b-bb64-4eea-986e-60e3f20da24a)